File: 27-remove-element/remove-element.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        i = 0
        while(nums.count(val)):
            nums.remove(val)
            i += 1
        return len(nums)

File: 27-remove-element/remove-element.c
int removeElement(int* nums, int numsSize, int val) {
    int* pointer = nums;
    int* end = nums + numsSize;
    while (pointer < end) {
        if (*pointer == val){
            end--;
            *pointer = *end;
        }
        else pointer++;
    }
    return end - nums;
}